src/server/routes/timeoffRequests.js

```diff
@@ -1,54 +1,11 @@
 const router = require('express').Router();
-const TimeoffRequests = require('../../models/timeoffRequests');
+const { processTimeOffRequest } = require('../../models/utilities');
 
-//test route
 router.post('/', (request, response) => {
-  const timeoffRequest = request.body.text;
-  console.log(timeoffRequest);
-
-  let message = {
-    "response_type": "ephemeral",
-    "text": "Your timeoff request was recorded!",
-    "attachments": [
-      {
-        "text": timeoffRequest
-      }
-    ]
-  };
-
-  response.json(message);
-
+  processTimeOffRequest(request.body)
+    .then(message => {
+      response.json(message)
+    })
```

Comment:
can you add a `catch` block here, and call `next(error)` so that the error handler middleware is triggered? If the error handler middleware does not exist, you may have to write one.